google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigtableOptions.java

```diff
@@ -134,10 +139,16 @@ public static Builder newBuilder() {
 
   static BigtableOptions fromPb(com.google.api.services.bigquery.model.BigtableOptions options) {
     Builder builder = newBuilder();
-    builder.setIgnoreUnspecifiedColumnFamilies(options.getIgnoreUnspecifiedColumnFamilies());
-    builder.setReadRowkeyAsString(options.getReadRowkeyAsString());
-    builder.setColumnFamilies(
-        Lists.transform(options.getColumnFamilies(), BigtableColumnFamily.FROM_PB_FUNCTION));
+    if (null != options.getIgnoreUnspecifiedColumnFamilies()) {
```

Comment:
reverse is more conventional; that is,

options != null

Reply:
It's unclear why null isn't allowed since that's the only reason to set Boolean instead of boolean

Reply:
Done.